docs/api-reference/io/index.rst

```diff
@@ -114,7 +114,7 @@ for new data:
    column of an output table.
 
 
-Serialization of Containers:
+Serialization of Containers
 ============================
```

Comment:
Need to also fix the underlining

Reply:
Done, that affected one further section that was also too long.